api-mssql-go/k8s/apirule.yaml

```diff
@@ -1,20 +1,39 @@
-apiVersion: gateway.kyma-project.io/v1beta1
+apiVersion: gateway.kyma-project.io/v2alpha1
 kind: APIRule
 metadata:
   name: api-mssql-go
 spec:
-  gateway: kyma-gateway.kyma-system.svc.cluster.local
-  host: api-mssql-go
+  hosts:
+    - api-mssql-go
   service:
     name: api-mssql-go
     port: 80
+  gateway: kyma-system/kyma-gateway
+  corsPolicy: 
+    allowHeaders:
+      - Content-Type
+    allowMethods:
+      - GET
+      - POST
+      - PUT
+      - DELETE
+      - OPTIONS
+    allowOrigins:
+      - regex: ".*{YOUR_DOMAIN}"
   rules:
-    - path: /orders|/orders/.*
-      accessStrategies:
-        - config: {}
-          handler: noop
+    - path: "/orders"
```

Comment:
could you please check with goats if regex can cover both path sections

Reply:
APIRule v2alpha1 doesn't support regex in paths. Please see https://kyma-project.io/#/api-gateway/user/custom-resources/apirule/04-70-changes-in-apirule-v2?id=path-specification-must-not-contain-regexp